cmd/lakectl/cmd/common_helpers.go

```diff
@@ -151,7 +151,7 @@ func WriteTo(tpl string, data interface{}, w io.Writer) {
 			var b strings.Builder
 			for _, row := range tab.Rows {
 				for ic, cell := range row {
-					b.WriteString(fmt.Sprintf("%s", cell))
+					b.WriteString(fmt.Sprintf("%v", cell))
```

Comment:
Can we use just fmt.Sprint? (Should we, given the danger of format during confusion?)

Reply:
I'll split the format and write. The Sprintf will render a single cell value - but in order to concat full table content I'm using strings.Builder.